src/openzaak/components/besluiten/api/validators.py

```diff
@@ -29,9 +29,7 @@ def __call__(self, attrs):
         if not zaak:
             return
 
-        if not besluittype.zaaktypes.filter(
-            id=zaak.zaaktype_id, concept=False
-        ).exists():
+        if not besluittype.zaaktypes.filter(id=zaak.zaaktype_id).exists():
```

Comment:
I don't understand this change?

When you're creating `besluit`, the `besluittype` must be published. If you are relating it to a `zaak`, the zaak must exist and have a published zaaktype. So, both besluittype and zaaktype should be published.

Or is that the logic you're using because the `concept` check actually doesn't matter here anymore?

Reply:
Yes, the idea was:
* make realization consistent with reference implementation realization, where there is no filter on `concept`
* get rid off excessive code: in this case we already know that `zaaktype` is published since related `zaak` exists